src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,99 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadlines extends Task {
```

Comment:
Might be better to name this class as `Deadline` instead as it is meant to represent a single deadline task